lib/routes/requests.js

```diff
@@ -0,0 +1,97 @@
+var Boom = require('boom')
+
+/* NOTES:
+  * What Auth do we use against CouchDB to get/set user doc? (user not logged in)
```

Comment:
You can copy how it's done in the `GET /session/account` handler here: https://github.com/hoodiehq/hapi-couchdb-account-api/blob/master/lib/routes/account.js#L28-L33

We do a straight request against couchDB, and set the cookie from the bearer token passed in the `Authorization` header. Note that the API does not expose the entire user doc, only id (not `_id` but a custom generated id that always remains the same, even if a username changes) and username. So you can safely add a `token` property to the _usersdoc.